package.json

```diff
@@ -30,7 +31,10 @@
     "@angular-devkit/build-angular": "^14.0.3",
     "@angular/cli": "~14.0.3",
     "@angular/compiler-cli": "^14.0.0",
+    "@types/googlemaps": "^3.43.3",
     "@types/jasmine": "~4.0.0",
+    "@types/jquery": "^3.5.14",
```

Comment:
We don't typically have a reason to import jquery in modern angular apps. The angular DOM manipulation typically happens in directives and angular provides ways to get DOM elements.